spec/controllers/dashboard/works_controller_spec.rb

```diff
@@ -14,7 +14,7 @@
   let(:invalid_attributes) {
     {
       'embargo_form' => {
-        'embargoed_until' => 'not a valid date'
+        'embargoed_until' => '2030-02-12'
```

Comment:
Could we do something like `Date.today + 5.years` here so it doesn't break in the future.  Assuming this is supposed to throw a validation error because of it being forward more than 4 years.